build.proj

```diff
@@ -23,6 +23,7 @@
     <Project Include="external\dir.proj" />
     <Project Include="src\ref.builds" />
     <Project Include="src\src.builds" />
+    <Project Include="src\tests.builds" />
```

Comment:
I thought we had decided not to add tests.builds as part of the main build any longer. We should either have a separate project for the tests, or at least condition it on a variable so that it is not built by default, and only built when build-tests.cmd/sh is called.